app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/Layout.scala.html

```diff
@@ -66,6 +69,18 @@
         <script src='@_root_.controllers.routes.Assets.versioned("javascripts/application.js")'></script>
     }
 
+    @hmrcStandardHeader = {
+
+        @hmrcHeader(Header(
+            homepageUrl = viewConfig.govUkUrl,
+            serviceName = Some(messages("service.title")),
+            serviceUrl = "https://www.gov.uk",
```

Comment:
what's the difference between `homepageUrl` and `serviceUrl`?

Reply:
One is a link to the gov.uk homepage, the other takes you to the start page of the service.

Reply:
At the moment you have them both pointing to the same url which can't be right.

Reply:
Just to add more info - I left serviceURL as its default as I don't think we have a URL for the start page yet, do we?

Reply:
Currently we use the default header which takes the message property `service.homePageUrl` and which we have set to `/claim-for-reimbursement-of-import-duties`. See

https://github.com/hmrc/play-frontend-govuk/blob/257bca399ca5c5b9fe7e30b18610ed1008dee8b8/src/main/twirl/uk/gov/hmrc/govukfrontend/views/layouts/govukLayout.scala.html#L44

Reply:
Cool - I can add in the property. I was looking for it in the viewConfig where most of the other links are defined.

Reply:
Using the service.homePageUrl property now - even though that link actually doesn't work yet, when it comes to testing just expect the same behaviour as the old header.

Reply:
This PR also fixes the feedback link URL - found it in the viewConfig when I was looking for the serviceURL :)